### PR TITLE
DOCS align docstring for trigonometric funcs with template

### PR DESCRIPTION
**Pull request by densmirn**

### CODE REVIEW

File: dpnp/backend_trigonometric.pyx [hunk 1]
```diff
@@ -177,6 +177,7 @@ cpdef dparray dpnp_unwrap(dparray array1):
         result = dparray(array1.shape)
 
     for i in range(result.size):
-        result[i] = numpy.unwrap(array1[i])
+        val, = numpy.unwrap([array1[i]])
+        result[i] = val
```
Comment (samir-nasibli) [inline dpnp/backend_trigonometric.pyx:181]: it is not good idea to mix "DOC docstrings align" with "development".
No changes needed in this PR. Please, separate such things in future.

Comment (densmirn) [reply dpnp/backend_trigonometric.pyx:181]: @samir-nasibli without the change the example in docstring doesn't work.

Comment (samir-nasibli) [reply dpnp/backend_trigonometric.pyx:181]: @densmirn therefore you need to make a separate PR to change the behavior of the function. In this case it is not so critical. Leave it as is. 
My comment was just an advice for the future.

Comment (densmirn) [reply dpnp/backend_trigonometric.pyx:181]: @samir-nasibli In general you are right, but in this case I prefer to keep this small fix to this PR if nobody minds.

**Pull request merged**